libensemble/tests/balsam_tests/readme.balsam_tests.txt

```diff
@@ -1,11 +1,11 @@
 List of balsam test jobs for libEnsemble.
 
 test_balsam_1__runjobs.py:
-Launches parallel parent job. Each task iterates over a loop launching sub-jobs
+Launches parallel parent job. Each job iterates over a loop launching sub-jobs
```

Comment:
Why jobs?

Reply:
Balsam launches "jobs", and these tests don't use the executor, let-alone libEnsemble.

Reply:
Yep, I think we can leave this alone as its just Balsam tests.